plat/kvm/Config.uk

```diff
@@ -29,6 +29,12 @@ config KVM_BOOT_PROTO_LXBOOT
 	help
 		Linux 64-bit Boot Protocol
 
+config KVM_BOOT_QEMU_VIRT
+	bool "QEMU Virt"
```

Comment:
Good change 👍🏼  Saying that, the machine name is `virt`, not `Virt`: https://qemu.readthedocs.io/en/latest/system/arm/virt.html. Also everywhere else, incl. the commit message.

Reply:
Thanks, but it's weird actually, this commit should have been higher up in the commit history, no idea what happened and how it ended up last.